kod/object/active/holder/nomoveon/battler/player/user.kod

```diff
@@ -8535,10 +8575,10 @@ messages:
          iFlags = iFlags | DRAWFX_INVISIBLE;
       }
 
-      if Send(self,@IsPhasedOut)
+      if Send(self,@IsInCannotInteractMode)
```

Comment:
We should keep the DRAWFX_DITHERINVIS effect for phase here, and use the DRAWFX_DITHERGREY effect for spectator mode. Easiest way would be to keep the original IsPhasedOut call, but after that add another check:

    if pbSpectatorMode
    {
       iFlags = iFlags & ~DRAWFX_MASK;
       iFlags = iFlags | DRAWFX_DITHERGREY;
    }